.circleci/config.yml

```diff
@@ -145,3 +148,6 @@ workflows:
       - benchmark:
           requires:
             - deploy
+          filters:
+            branches:
+              only: /^[^:]+$/
```

Comment:
Might be good to have a comment there, it is not entirely obvious. I assume it looks for `:` in the branch name, right?